src/service/extension-location.js

```diff
@@ -49,13 +49,19 @@ export function calculateExtensionScriptUrl(location, extensionId, isLocalDev) {
  * @param {string} entryPoint
  * @param {boolean=} isLocalDev
  * @param {boolean=} opt_rtv
+ * @param {boolean=} opt_minified
  * @return {string}
  */
 export function calculateEntryPointScriptUrl(
-    location, entryPoint, isLocalDev, opt_rtv) {
+    location, entryPoint, isLocalDev, opt_rtv, opt_minified) {
```

Comment:
> Can't you set the test server to serve max files?

Hmm, we could do that. We'd need to dupe the logic in `app.js` into `test-server.js`.

> I thought we always serve unminified version when running gulp test without --compiled

Only for the `app.js` server which currently doesn't run during testing. 

Another issue is that `location.host` actually returns the Karma server on localhost, so that needs to change to the test server's port.